cylc/flow/scheduler.py

```diff
@@ -909,18 +913,43 @@ def process_queued_task_messages(self) -> None:
                     should_poll = True
             if should_poll:
                 to_poll_tasks.append(itask)
-        self.task_job_mgr.poll_task_jobs(
-            self.workflow, to_poll_tasks)
+        if to_poll_tasks:
+            self.task_job_mgr.poll_task_jobs(self.workflow, to_poll_tasks)
+        # Remaining messages don't have a receiving task in the pool.
+        # E.g., after manually setting a running task to finished.
+        for _id, tms in messages.items():
+            warn = "Undeliverable task messages received and ignored:\n"
+            for _, msg in tms:
+                warn += f'{msg.job_id}: {msg.severity} - "{msg.message}"'
```

Comment:
These will all end up on the same line
```suggestion
            warn = "Undeliverable task messages received and ignored:"
            for _, msg in tms:
                warn += f'\n  {msg.job_id}: {msg.severity} - "{msg.message}"'
```